cpp/oneapi/dal/test/engine/dataframe_common.cpp

```diff
@@ -344,25 +381,75 @@ static homogen_table build_homogen_table(Policy& policy,
 }
 
 template <typename Policy>
-static table build_table(Policy& policy, const dataframe& df, const table_id& id) {
+static homogen_table build_homogen_table(Policy& policy,
+                                         const array<float>& data,
+                                         const table_id& id,
+                                         std::int64_t row_count,
+                                         std::int64_t first_column,
+                                         std::int64_t last_column) {
+    const std::int64_t data_column_count = data.get_count() / row_count;
```

Comment:
Recommend to pass `column_count` as separate parameter

Reply:
This is not addressed.

Reply:
sorry